README.rst

```diff
@@ -19,12 +19,12 @@ The Codelists
 
 The source codelists can be found in the `xml/` directory. 
 
-Embedded vs NonEmbedded Codelists
+Core vs NonCore Codelists
```

Comment:
Is there a certain naming convention for NonCore? i.e **Non-Core** vs **NonCore** @Ocre42 @amy-silcock